tests/DfE.FindInformationAcademiesTrusts.UnitTests/Pages/CookiesModelTests.cs

```diff
@@ -0,0 +1,57 @@
+using DfE.FindInformationAcademiesTrusts.Pages;
+using Microsoft.AspNetCore.Http;
+using Microsoft.AspNetCore.Mvc;
+using Microsoft.AspNetCore.Mvc.RazorPages;
+
+namespace DfE.FindInformationAcademiesTrusts.UnitTests.Pages;
+
+public class CookiesModelTests
+{
+    private readonly DefaultHttpContext _httpContext;
+    private readonly Mock<IResponseCookies> _mockResponseCookies;
+    private readonly Mock<ISession> _mockSession;
+    private readonly CookiesModel _sut;
+    private readonly Mock<IHttpContextAccessor> _mockHttpAccessor;
+
+
+    public CookiesModelTests()
+    {
+        _mockHttpAccessor = new Mock<IHttpContextAccessor>();
+        _httpContext = new DefaultHttpContext();
+        _mockResponseCookies = new Mock<IResponseCookies>();
+        _mockSession = new Mock<ISession>();
+        _mockHttpAccessor.Setup(m => m.HttpContext).Returns(_httpContext);
+        _mockHttpAccessor.Setup(m => m.HttpContext!.Session).Returns(_mockSession.Object);
+        _mockHttpAccessor.Setup(m => m.HttpContext!.Response.Cookies).Returns(_mockResponseCookies.Object);
+        _sut = new CookiesModel(_mockHttpAccessor.Object);
+    }
+
+    [Theory]
+    [InlineData("page")]
+    [InlineData("")]
+    [InlineData(null)]
+    public void Visit_cookies_page_from_footer(string page)
```

Comment:
Please could we make all of these unit test names more descriptive? e.g. `OnGetAsync_should_show_page_if_directly_navigated_to_from_another_page`